### PR TITLE
Multi-experiment sketch

### PR DESCRIPTION
**Pull request by czue**
This change introduces a parent/child experiment relationship configured via the `ExperimentRoute` model. This allows you to create a top-levell "router" bot that then routes to other bots underneath it, based on a keyword. I created three bots with the instructions:

```
Respond to the user's input, but always respond in {LANGUAGE} no matter the history of the conversation.
```

Then I created a router with this modified prompt that I got from @bderenzi

```
You are a router, tasked with routing messages from a user to one of three workers: ENGLISH, SPANISH,
and FRENCH. You must respond only with the name of one of these routers. ENGLISH is your default answer.
If the user wishes to discuss anything related to food (or is already talking about food), then route to FRENCH.
If the user wishes to discuss computers (or anything related to computers), route to SPANISH. For context, you
will be given a the previous five messages back and forth between the workers and the user. If you are given
less than that it is because the conversation has just started.
```

And then I set up three routes in the Django admin:

image

And I can have the following conversation:

image

### CODE REVIEW

**Comment by czue** [comment]
@snopoke I think this is ready now

**Comment by czue** [comment]
Oh actually I need to update token counting.

**Comment by czue** [comment]
> Oh actually I need to update token counting.

Done

**Review by snopoke — APPROVED** [review]

File: apps/experiments/models.py [hunk 1]
```diff
@@ -402,6 +405,23 @@ def get_absolute_url(self):
...
 
 
+class ExperimentRoute(BaseTeamModel):
+    """
+    Through model for Experiment.children routes.
+    """
+
+    parent = models.ForeignKey(Experiment, on_delete=models.CASCADE, related_name="child_links")
+    child = models.ForeignKey(Experiment, on_delete=models.CASCADE, related_name="parent_links")
+    keyword = models.CharField(max_length=128)
```
[the remark this answers is not in the document]
Comment (czue) [reply apps/experiments/models.py]: good call 3fa935e48c17f97396665ab884d9be1a947b67d1

**Pull request merged**